connector/profilingmetricsconnector/connector_test.go

```diff
@@ -282,40 +264,11 @@ func TestConsumeProfiles_NoMetrics(t *testing.T) {
 	// Expect ConsumeMetrics NOT to be called.
 	err := conn.ConsumeProfiles(context.Background(), profiles)
 	assert.NoError(t, err)
-	mockConsumer.AssertNotCalled(t, "ConsumeMetrics", mock.Anything, mock.Anything)
-}
-
-func TestConsumeProfiles_ConsumeMetricsError(t *testing.T) {
```

Comment:
This is not needed atm, the actual `ConsumeMetrics` implementation cannot return an error.